agent_benchmark_runner.py

```diff
@@ -160,7 +163,7 @@ def close(self):
             try:
                 self._progress_csv_file.flush()
                 os.fsync(self._progress_csv_file.fileno())
-            except Exception:
+            except IOError:
```

Comment:
Fix a broad exception. Some of these didn't have enough context to know what exception was really coming.